plugin/crystalline.vim

```diff
@@ -18,27 +18,6 @@ let g:crystalline_mode_hi_groups = {
       \ '': '',
       \ }
 
-if get(g:, 'crystalline_enable_sep', 0)
```

Comment:
I believe there is also documentation for this to remove

Reply:
Done.